INSTALL.rst

```diff
@@ -57,17 +57,19 @@ After installing GAMS, we recommend that new users install Anaconda, and then us
 Advanced users may choose to install |MESSAGEix| from source code (next section).
 
 4. Install Python via either `Miniconda`_ or `Anaconda`_. [1]_
-   We recommend the latest version; currently Python 3.7.
+   We recommend the latest version; currently Python 3.8.
 
 5. Open a command prompt.
-   Windows users should use the “Anaconda Prompt” to avoid issues with permissions and environment variables when installing and using |MESSAGEix|.
+   **Windows users should use the “Anaconda Prompt”** to avoid issues with permissions and environment variables when installing and using |MESSAGEix|.
    This program is available in the Windows Start menu after installing Anaconda.
 
 6. Configure conda to install :mod:`message_ix` from the conda-forge channel [2]_::
 
     $ conda config --prepend channels conda-forge
 
-7. Create a new conda enviroment.
+Alternatively to *Steps 5. and 6.*, :mod:`message_ix` can also be installed using the **“Anaconda Navigator”** (see instructions `here`_)
+
+7. Create a new conda environment.
    This step is **required** if using Anaconda, and *optional* if using Miniconda.
    This example uses the name ``message``, but you can use any name of your choice::
```

Comment:
The instructions to create a new conda environment worked for me, but it would be useful to also update the instructions for running tutorials which then require making sure that Jupyter uses this environment when running the tutorials or any other ixmp/message_ix code. There are again different ways of doing this, either starting Jupyter from command line after activating the environment (@adrivinca is using this method) or via Anaconda Navigator which allows selecting an environment in which to deploy applications including Jupyter.